bigtable/google/cloud/bigtable/table.py

```diff
@@ -29,6 +29,14 @@
 from google.cloud.bigtable.row_data import PartialRowsData
 
 
+class RowBelongingError(Exception):
+    """Row from another table."""
+
+
+class TooManyMutationsError(Exception):
+    """The number of mutations for bulk request is too big."""
```

Comment:
In general I (and this library) is/are :-1: on custom exceptions. For example, this one would be just fine as a `ValueError`.